lib/src/barback/dart2js_transformer.dart

```diff
@@ -280,7 +281,7 @@ class _BarbackCompilerProvider implements dart.CompilerProvider {
   }
 
   /// A [CompilerInputProvider] for dart2js.
-  Future<String> provideInput(Uri resourceUri) {
+  Future provideInput(Uri resourceUri) {
```

Comment:
I'd write `Future/*<String | List<int>>*/`

Reply:
Done (I had to add the extra space to avoid an analyzer warning about generic comment syntax, though)